src/launch/launch.sh

```diff
@@ -14,8 +14,12 @@ if [[ ! $(microstack.openstack keypair list | grep "| microstack |") ]]; then
     mkdir -p $HOME/.ssh
     chmod 700 $HOME/.ssh
     microstack.openstack keypair create microstack >> $HOME/.ssh/id_microstack
+    chmod 600 $HOME/.ssh/id_microstack
 fi
 
+echo "Launching instance ..."
```

Comment:
I moved this above the creation of security groups, as changes to the default group will take effect, even for a running instance!